src/atoms/linePager/component.tsx

```diff
@@ -58,7 +60,11 @@ interface ILinePagerProps extends IComponentProps<ILinePagerTheme> {
   onPageClicked?(index: number): void;
 }
 
-export const LinePager = (props: ILinePagerProps): React.ReactElement => {
+export const LinePager = (inputProps: ILinePagerProps): React.ReactElement => {
+  const props = setDefaults(inputProps, {
+    ...defaultComponentProps,
+
```

Comment:
lots of these have this extra line - it should not be here please remove anywhere you have added it. There should never be extra lines inside an object definition.